ui/src/shared/graphs/helpers.js

```diff
@@ -51,24 +51,32 @@ export const multiColumnBarPlotter = e => {
     }
   }
 
-  const barWidth = Math.floor(2.0 / 3 * minSep)
+  const barWidth = Math.max(Math.floor(2.0 / 3.0 * minSep), 1)
 
   const fillColors = []
   const strokeColors = g.getColors()
+
+  let selPointX
+  if (g.selPoints_ && g.selPoints_.length) {
+    selPointX = g.selPoints_[0].canvasx
+  }
+
   for (let i = 0; i < strokeColors.length; i++) {
     fillColors.push(darkenColor(strokeColors[i]))
   }
 
+  ctx.lineWidth = 2
+
   for (let j = 0; j < sets.length; j++) {
-    ctx.fillStyle = fillColors[j]
     ctx.strokeStyle = strokeColors[j]
     for (let i = 0; i < sets[j].length; i++) {
       const p = sets[j][i]
       const centerX = p.canvasx
+      ctx.fillStyle = fillColors[j]
       const xLeft =
         sets.length === 1
-          ? centerX - barWidth / 2
-          : centerX - barWidth / 2 * (1 - j / (sets.length - 1))
+          ? centerX - barWidth / 1
```

Comment:
haha, probably a brain fart, just due to experimentation
thanks for pointing that out!